contrib/python-bindings/tests/triangulation_wrapper.py

```diff
@@ -375,6 +380,31 @@ def test_refine_global(self):
             else:
                 self.assertEqual(n_cells, 64)
 
+
+    def test_transform(self):
+        for dim in self.dim:
+            triangulation_1 = self.build_hyper_cube_triangulation(dim)
+            triangulation_1.refine_global(1)
+            triangulation_2 = self.build_hyper_cube_triangulation(dim)
+            triangulation_2.refine_global(1)
+
+            triangulation_1.transform(lambda p: [v+1. for v in p])
+            triangulation_1.transform(lambda p: [v-1. for v in p])
+
+            for (cell_1, cell_2) in zip(triangulation_1.active_cells(), triangulation_2.active_cells()):
+                self.assertTrue(cell_1.center().distance(cell_2.center()) < 1e-8)
```

Comment:
This test is really weak because if `transform` does nothing the test will pass. What about removing the second `transform` and checking that `cell_1` is just `cell_2` displaced by **1**